### ISSUE TITLE
Update Warning and Note alert/banner colors

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/component-library#193]
SODA reports some color contrast violations with our alert/banner icon colors.

Originally, we considered these icons "decorative" images, and thus not requiring a 3:1 contrast ratio. However, in practice we sometimes use these alerts/banners without their header text ("Warning", "Note", etc.), so the icons communicate information and thus are non-decorative.

To satisfy contrast requirements, we need to make the following changes. In both [Alerts](https://sul-dlss.github.io/component-library/alerts/) and [Banner](https://sul-dlss.github.io/component-library/banners/), do the following:

- For the "Warning" (orange) alert/banner, change the color from `--stanford-poppy-rgb` (233, 131, 0) to `--stanford-poppy-dark-rgb` (209, 102, 15).
- For the "Note" (yellow) alert/banner: change the color from `--stanford-illuminating-dark-rgb` (254, 197, 29) to `--stanford-plum-light-rgb` (115, 70, 117).

This may require adding the set of `stanford-plum` colors to the component library:
`--stanford-plum-rgb` (98, 0, 89) // #620059
`--stanford-plum-light-rgb` (115, 70, 117) // #734675
`--stanford-plum-dark-rgb` (53, 13, 54) // #350D36
https://identity.stanford.edu/design-elements/color/accent-colors/

### Reported by SODA

> [Summary]
> An icon lacks 3 to 1 contrast ratio
> 
> [Description]
> Parts of an icon (with no text) do not have a contrast ratio of 3 to 1 against adjacent color(s). These icon parts are required for understanding.
> 
> Alert icon has a contrast ratio of 2.5:1 which is less than the required 3:1. 
> Screenshot: https://stanford-axeauditor.dequecloud.com/api/v1/file/760e05be-835b-11f0-8add-2be00a3caac0
> Foreground: #E98300
> Background: #FDF2E5
> Contrast ratio: 2.465:1
> 
> Alert icon has a contrast ratio of 1.6:1 which is less than the required 3:1. 
> Screenshot: https://stanford-axeauditor.dequecloud.com/api/v1/file/62507d12-8352-11f0-8eb9-db4dd20d8c8c
> Foreground: #FAC51D
> Background: #FFFFFF
> Contrast ratio: 1.607:1
> 
> [Impact]
> Serious
> 
> [Recommended to fix]
> Rule : 
> Parts of graphics (required to understand the content) MUST have a contrast ratio of 3 to 1 against adjacent color(s). Exceptions exist.
> 
> How To Fix: 
> Fix this issue by adjusting the parts of graphics (that are required to understand the content) and/or adjacent color to increase the contrast to at least 3 to 1.
> 
> Reference: 
> WCAG Understanding document: https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast
> 
> Background: 
> People who have low vision or are colorblind may have difficulty perceiving the important parts of an informative graphical object (i.e., the parts of the graphic required to understand the content) if the contrast between parts of the graphic essential for understanding the graphic is insufficient. When an graphical object has adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive the information conveyed by the object.
> NOTE: The term "graphical object" applies to stand-alone icons such as a print icon (with no text), and the important parts of a more complex diagram such as each line in a graph. For simple graphics such as single-color icons the entire image is a graphical object. Images made up of multiple lines, colors and shapes will be made of multiple graphical objects, some of which are required for understanding. Not every graphical object needs to contrast with its surroundings - only those that are required for a user to understand what the graphic is conveying. This requirement does not apply to a graphic whose information is available in another form such as text - for example a printer icon with the word "print" next to it - and it does not apply to decorative images that do not convey information.
>
